library/globals.inc.php

```diff
@@ -545,6 +545,40 @@ function gblTimeZones()
             xl('Changes the layout of the login page.')
         ),
 
+        'primary_logo_width' => [
+            xl('Width of logo compared to the container'),
+            [
+                'w-25' => '25%',
+                'w-50' => '50%',
+                'w-75' => '75%',
+                'w-100' => '100%'
+            ],
+            'w-50',
+            xl('Determine the width of the primary logo compared to the container'),
+        ],
+
+        'secondary_logo_width' => [
+            xl('Width of secondary compared to the container'),
```

Comment:
is logo supposed to be in the title?
Width of secondary logo compared to the container